.github/workflows/release.yml

```diff
@@ -41,6 +45,8 @@ jobs:
           _output/release/${{ matrix.target }}-${{ matrix.os }}-${{ matrix.arch }}.tgz
           _output/release/${{ matrix.target }}-${{ matrix.os }}-${{ matrix.arch }}.tgz.sha256
   release-crds-assests:
+    permissions:
+      contents: write  # for softprops/action-gh-release to create GitHub release
```

Comment:
following the doc,set the correct permission.  https://github.com/softprops/action-gh-release?tab=readme-ov-file#permissions
/lgtm